configs/faster_rcnn/README.md

```diff
@@ -38,7 +38,7 @@ We trained with R-50-FPN pytorch style backbone for 1x schedule.
 | :------: | :------------: | :------: | :------------: | :----: | :--------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | R-50-FPN |     L1Loss     |   4.0    |      21.4      |  37.4  |   [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py)   |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_1x_coco_20200130_204655.log.json)             |
 | R-50-FPN |    IoULoss     |          |                |  37.9  | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn/faster-rcnn_r50_fpn_iou_1x_coco.py) | [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_iou_1x_coco/faster_rcnn_r50_fpn_iou_1x_coco_20200506_095954-938e81f0.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_iou_1x_coco/faster_rcnn_r50_fpn_iou_1x_coco_20200506_095954.log.json) |
-| R-50-FPN |    GIoULoss    |          |                |  37.6  |   [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py)   |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco-0eada910.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco_20200505_161120.log.json)            |
+| R-50-FPN |    GIoULoss    |          |                |  37.6  |  [config](https://github.com/open-mmlab/mmdetection/blob/3.x/configs/faster_rcnn/faster-rcnn_r50_fpn_giou_1x_coco.py)  |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco-0eada910.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco_20200505_161120.log.json)            |
```

Comment:
```suggestion
| R-50-FPN |    GIoULoss    |          |                |  37.6  |  [config](./faster-rcnn_r50_fpn_giou_1x_coco.py)  |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco-0eada910.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco_20200505_161120.log.json)            |
```

Reply:
I submitted your suggestion last time, but could not pass the lint test